docs/scaffold-checklist.md

```diff
@@ -23,8 +23,10 @@ Then double check your work with those helper scripts checks:
 - `yarn run semver` check passes
 
 **Important!** Make your new version available in the upgrade tool (npx crowdbotics/modules) by updating [config.js](/config.js) to include a new `config.upgrade.versions` and then run:
+in order to bump the version, we need to modify `package.json` at `scaffold/package.json` and `.crowdbotics.json` at `scaffold/template/custom/.crowdbotics.json`
```

Comment:
```suggestion
**Important!** Increase the version of the scaffold in [package.json](/scaffold/package.json) and [.crowdbotics.json](/scaffold/template/custom/.crowdbotics.json).

Make this new version available in the upgrade tool (npx crowdbotics/modules) by updating [config.js](/config.js) to include a new `config.upgrade.versions` and then finally run:
```
Suggestion: Move the package.json sentence to go above (since users have to do that first) and also so that the `run:` part is followed up by the command.